### ISSUE TITLE
Activity Tab -- Progress from download client on each episode is reported per torrent's overall progress, instead of per episode

### ISSUE THREAD

**Issue by wikid24** [Sonarr/Sonarr#7044]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Scenario: Downloading a season's worth of episodes from a single torrent from download client (qBittorrent). 

Sonarr is tracking each episode individually on the activity tab. It reports each episode's progress %. Every episode has the same % progress bar. 

Check the qbittorrent progress for each file, and it shows the the download progress of each episode

![image](https://github.com/user-attachments/assets/5275dbb3-d9d4-42c0-b41b-3e5124a34beb)

### Expected Behavior

Progress of each file tracked individually on Sonarr's activity tab. 

Kind of a low priority, I just thought I'd report it since I noticed it this morning :)

Thanks!

### Steps To Reproduce

1) In torrent download client, find a torrent with a full season's worth of episodes on it. 
2) set file priroity in download client so that some files download faster than others
3) make sure that sonarr is monitoring for multiple episodes and can track progress on the activity tab

### Environment

```markdown
- OS: PI Os
- Sonarr: 4.0.8.1874
```

### What branch are you running?

Main

### Trace Logs?

None required

**Comment by PearsonFlyer** [comment]
When a season pack is grabbed, the progress is for the entire pack, since no episodes will be imported until the entire torrent is complete. This is expected behavior. Qbit does not report the status of each file inside a torrent.

**Comment by wikid24** [comment]
Ah ok I didn't know that Sonarr waits for the entire torrent to finish before starting the import process. Thanks for the head's up!
